services/permission.go

```diff
@@ -44,22 +44,22 @@ func (ps *Permission) GetForServer(serverId string) ([]*models.Permissions, erro
 	return *allPerms, err
 }
 
-func (ps *Permission) GetForUserAndServer(userId uint, serverId *string) (*models.Permissions, error) {
+func (ps *Permission) GetForUserAndServer(userId int, serverId *string) (*models.Permissions, error) {
 	permissions := &models.Permissions{
 		UserId:           &userId,
 		ServerIdentifier: serverId,
 	}
 
 	err := ps.DB.Preload("User").Preload("Server").Where(permissions).First(permissions).Error
 
-	if err != nil && gorm.IsRecordNotFoundError(err) {
+	if err != nil && gorm.ErrRecordNotFound == err {
```

Comment:
https://gorm.io/docs/v2_release_note.html#ErrRecordNotFound says it should be done as a errors.Is